client/my-sites/domains/domain-management/dns/dns-records-list.jsx

```diff
@@ -196,10 +196,15 @@ class DnsRecordsList extends Component {
 			];
 		}
 
-		return [
-			{ ...this.editRecordAction, disabled: record.protected_field },
-			{ ...this.deleteRecordAction, disabled: record.protected_field && 'MX' !== record.type },
-		];
+		const actions = [];
+
+		if ( ! record.protected_field ) {
+			actions.push( { ...this.editRecordAction, disabled: false } );
+		}
+		if ( ! ( record.protected_field && 'MX' !== record.type ) ) {
+			actions.push( { ...this.deleteRecordAction, disabled: false } );
```

Comment:
Since we're always just returning `false` for the `disabled` property here now, I guess we can probably remove this property and remove the checks for it above. Does that seem right?